packages/playground/src/components/widgets/widget-connection/widget-connection.tsx

```diff
@@ -18,6 +18,7 @@ export class WidgetConnection {
   @Prop() network: string = "";
 
   render() {
+    console.log("network", this.network)
```

Comment:
Is this console.log needed?

Reply:
Definitely not. Just took it out in the latest commit.